src/main/java/com/pirimid/cryptotrade/controller/AccountController.java

```diff
@@ -0,0 +1,40 @@
+package com.pirimid.cryptotrade.controller;
+
+import com.pirimid.cryptotrade.model.Account;
+import com.pirimid.cryptotrade.model.Order;
+import com.pirimid.cryptotrade.service.AccountService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+import java.util.UUID;
+
+@RestController
+public class AccountController {
+    @Autowired
+    private AccountService accountService;
+    @GetMapping("/accounts")    // accounts?exchange=
+    public ResponseEntity<Set<Account>> getAccountsByExchange(@RequestParam String exchange){
```

Comment:
In response it's good practice to return page instance rather than sending every data

Reply:
For now We haven't created ModelDTOs. We were primarily working on Exchange part so now onwards We will work on it.